spec/models/factories_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'spec_helper'
```

Comment:
Since you have `--require spec_helper` in your `.rspec` file, you don't need to require it in your tests. Remove it from here and the others